.github/ISSUE_TEMPLATE/rfc-template.md

```diff
@@ -0,0 +1,112 @@
+---
+name: RFC Template
+about: Fill in this template and you are on your way to submit an RFC to the Project-Origin
+  working group.
+title: "[RFC]000 - Issue title"
+labels: rfc
+assignees: ''
+
+---
+
+---
```

Comment:
This looks sort of duplicated. Did you verify both metadata sections are required?

Reply:
I believe these metadata are for [Issue templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository) on github - that is why they are duplicated, consider them a markdown header :-)